proto/xudrpc.proto

```diff
@@ -42,9 +42,9 @@ service Xud {
   }
 
   /* Cancel placed order from the orderbook. */
```

Comment:
Can you update this comment as well? I think just something simple like "Removes an order from the order book by its local id." would be fine for now.